doc/reference.rst

```diff
@@ -42,7 +42,7 @@ either to read voltage at ADC pin, or to read module supply voltage
 (VCC).
 
 To read external voltage applied to ADC pin, use ``analogRead(A0)``.
-Input voltage range is 0 — 1.0V.
+Input voltage range is 0 — 1.0V or 0 — 3.3V.
 
 To read VCC voltage, use ``ESP.getVcc()`` and ADC pin must be kept
```

Comment:
There also should be mentioned, that many boards (it means basicaly all boards which are not bare esp) has voltage divedr in theirs design, so ESP.getVcc() will produce totaly wrong results - possible resistor connected to A0 pin has to be removed before using this function